docs/docs/getting_started/noir_installation.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Alternative Installations
+title: Standalone Noir Installation
```

Comment:
```suggestion
title: Advanced Noir Installation
```

"Standalone" doesn't seem very intuitive for what this page covers. Maybe "Advanced"?

Reply:
We could consider moving this page to another section / standalone as well

Reply:
We want `noirup` as the standard way to install Nargo, so I wouldn't call it "advanced". Just "Installation" would be confusing though specially as it mostly a reference.

I'll think about other options and iterate later on

Reply:
> We want noirup as the standard way to install Nargo, so I wouldn't call it "advanced".

I would consider `noirup` and `noirup -v ...` to be the common installation path, where all other `noirup ...` installation paths to be advanced usage.